### PR TITLE
[move] Update why move

### PR DESCRIPTION
**Pull request by gregnazario**
### Description
This adds some more detail / cleanups to some grammar

### Checklist

- Do all Lints pass?
  - [ ] Have you ran `pnpm spellcheck`?
  - [x] Have you ran `pnpm fmt`?
  - [x] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/nextra/pages/en/build/smart-contracts/why-move.mdx [hunk 1]
```diff
@@ -4,90 +4,144 @@ title: "Why Move?"
...
+digital assets on-chain.  These were influenced by existing standards such as
+ERC-20, ERC-721, ERC-1155 and Token-2022.
 
-8. **Token Types**: Like [Token-2022](https://spl.solana.com/token-2022), Fungible Asset and Digital Asset cover the basis in what is possible within the token space without introducing additional token types.
+8. **On-chain Randomness**: Native on-chain unbiasable randomness provides a safe
+and consistent way of getting random numbers, with extra safety checks at compile
+time.
 
 
 ## Comparison to Other Blockchains
```
Comment (wrwg) [inline apps/nextra/pages/en/build/smart-contracts/why-move.mdx:98]: I thinks its fine right now, but do not promise anything about not having dynamic dispatch (no re-entrency is fine though)

File: apps/nextra/pages/en/build/smart-contracts/why-move.mdx [hunk 2]
```diff
@@ -4,90 +4,144 @@ title: "Why Move?"
...
+specifically to provide formal verification of contracts.  This allows for proving
+invariants and as a tool to help code auditing.
+
+4. **Simplicity**: The commands and bytecode are purposely simple.  This allows
+for easy decompilation, runtime verification, and code inspection.  More complex
+languages often have whole parts that need to be ignored e.g. Rust.
+
+# Why Move on Aptos?
+
+Move on Aptos supports the full language built by the team at Facebook, with some
```
Comment (wrwg) [inline apps/nextra/pages/en/build/smart-contracts/why-move.mdx:38]: s/some additional/additional

**Review by wrwg — COMMENTED** [review]
LGTM

File: apps/nextra/pages/en/build/smart-contracts/why-move.mdx [hunk 3]
```diff
@@ -4,90 +4,92 @@ title: "Why Move?"
...
+The Move programming language was originally created by a team of engineers at
+Facebook for the Diem Payment Network. Move is designed to be a platform-agnostic
+language to enable common libraries, tooling, and developer communities across
+diverse blockchains with vastly different data and execution models.
+At Aptos, we believe in building a strong developer community in Move and invite
+them to build upon the Move on Aptos stack and contribute to the open source software.
 
+Move was built upon the idea that other programming languages made it harder to
+provide guarantees at runtime, which is required for a trusted execution
+environment.
```
Comment (hariria) [inline apps/nextra/pages/en/build/smart-contracts/why-move.mdx:16]: I'm a bit confused by what you mean here. Some clarity or examples on the guarantees might help. Maybe something like:

```suggestion
Move was designed to provide safety and security guarantees at runtime, a novel idea not implemented by
other smart contract languages.
```

Probably want to add something else after

File: apps/nextra/pages/en/build/smart-contracts/why-move.mdx [hunk 4]
```diff
@@ -4,90 +4,92 @@ title: "Why Move?"
...
+reentrancy attacks, double spends, and arithmetic overflow.  Type safety and
+compile time checks are at the forefront of the security.
 
-1. **Formal Verification**: Aptos framework is fully specified and formally verified. This includes core contracts including governance, NFTs, and Tokens.
+2. **Runtime Verification**: The bytecode can be verified at runtime to verify
+that nothing has gone wrong, providing extra safety and preventing malicious actors.
 
-2. **Gas Coverage**: Move VM has 100% gas coverage. Gas is charged based upon actual usage in the system (CPU, memory, storage, I/O). In other words, no simple free gas exploits.
+3. **Formal Verification**: The language provides an extension to the Move language
+specifically to provide formal verification of contracts.  This allows for proving
```
Comment (hariria) [inline apps/nextra/pages/en/build/smart-contracts/why-move.mdx:29]:
```suggestion
3. **Formal Verification**: Move on Aptos includes an extension to the language 
-- the Move Prover -- to provide formal verification of contracts.  This allows for proving
```

File: apps/nextra/pages/en/build/smart-contracts/why-move.mdx [hunk 5]
```diff
@@ -4,90 +4,92 @@ title: "Why Move?"
...
+invariants and as a tool to help code auditing.
```
Comment (hariria) [inline apps/nextra/pages/en/build/smart-contracts/why-move.mdx:30]:
```suggestion
invariants and helps with code auditing.
```

File: apps/nextra/pages/en/build/smart-contracts/why-move.mdx [hunk 6]
```diff
@@ -4,90 +4,92 @@ title: "Why Move?"
...
 
+Move on Aptos supports the full language built by the team at Facebook, with
+additional extensions built to improve the security and the developer experience.
 
-## Developer Experience (DevX)
+## Security
 
-1. **Tools Suite**: Aptos has a suite of tools developers expect beyond the compiler: decompiler, formatter, analyzer, disassembler, specification tester, CLI.
+1. **Formal Verification**: Aptos framework is fully specified and formally
+verified. This includes the core contracts involving governance, NFTs, and Tokens.
```
Comment (hariria) [inline apps/nextra/pages/en/build/smart-contracts/why-move.mdx:45]:
```suggestion
verified via the Move Prover. This includes the core contracts involving governance, NFTs, and Tokens.
```

File: apps/nextra/pages/en/build/smart-contracts/why-move.mdx [hunk 7]
```diff
@@ -4,90 +4,92 @@ title: "Why Move?"
...
+2. **Gas Coverage**: Move VM has 100% gas coverage. Gas is charged based upon
+actual usage in the system (CPU, memory, storage, I/O). In other words, no
+gas exploits.
 
-3. **Data Model**: Aptos has a more accessible data model with the data definition stored on-chain and objects and accounts can have multiple distinct structures in an easy-to-parse format.
+3. **Security Redundancy**: Security redundancy provided by runtime safety checks.
 
-4. **Safe Upgradeability**: Unique safe upgradeability ensures that application interfaces cannot be broken and doesn't require explicit adoption from downstream applications.
+4. **Permission Controls**: Permission controls can flexibly be built at various
+levels.  For example, at the token level permission controls are by default to
```
Comment (hariria) [inline apps/nextra/pages/en/build/smart-contracts/why-move.mdx:54]: I think you're missing something here. `...are __blank__ by default...`

File: apps/nextra/pages/en/build/smart-contracts/why-move.mdx [hunk 8]
```diff
@@ -4,90 +4,92 @@ title: "Why Move?"
...
+enable RWA tokenization.
 
-5. **Cross-Interaction**: Move allows for more intuitive cross (module, contract, program) interaction by using type-safe structs.
 
-6. **Code Storage**: Aptos stores code on-chain improving the ability to audit and ensure contract to byte code correctness.
+## Developer Experience
 
-7. **Sponsored Transactions**: Only application with native sponsored transaction support — no special services or contract-specific code required.
+1. **Move Development Tools**:
+  - **Unit testing**: Move has built in unit testing for all contracts.  Aptos
```
Comment (hariria) [inline apps/nextra/pages/en/build/smart-contracts/why-move.mdx:61]:
```suggestion
  - **Unit testing**: Move has built-in unit testing for all contracts.  Aptos
```

File: apps/nextra/pages/en/build/smart-contracts/why-move.mdx [hunk 9]
```diff
@@ -4,90 +4,92 @@ title: "Why Move?"
...
+additionally, provides test functionality in the framework to test different
```
Comment (hariria) [inline apps/nextra/pages/en/build/smart-contracts/why-move.mdx:62]:
```suggestion
additionally provides test functionality in the framework to test different
```

**Review by hariria — APPROVED** [review]

File: apps/nextra/pages/en/build/smart-contracts/why-move.mdx (hunk 3 above)
Comment (gregnazario) [reply apps/nextra/pages/en/build/smart-contracts/why-move.mdx:16]: I'm removing this sentence

**Pull request merged**